CONTRIBUTING.md

```diff
@@ -23,6 +23,105 @@ First of all, thank you for contributing. It’s appreciated.
 
 ## Component structure
 
+### Create a new Component
+Here are some basic steps to get you started building your first component.
+
+1. Create your new component folder
+`src/components/MyNewComponent/`
+2. Create your component file(s)
+```
+    /MyNewComponent/
+      |-examples/
+        |-0.default.jsx <-- default example, used by `gulp docs-generate` to create component demo
+      |-MyNewComponent.jsx  <-- component logic goes here
+      |-MyNewComponent/MyNewComponent.less <-- component styling goes here
+      |-MyNewComponent/MyNewComponent.spec.js <-- component tests go here
+  ```
+3. Component Template
```

Comment:
I like the idea of having a template for a component
Have you considered making this an actual .jsx file in the repo?
that would allow it to be linted like the rest of the files
and also it might be useful for people to see what it looks like in the demo page
and play around with it and modify it for practice/experimentation
You could easily just not export it in index.js